config/app.default.php

```diff
@@ -408,4 +408,18 @@
             'users',
         ],
     ],
+
+    /**
+     * Additional plugins to load with this format: 'PluginName' => load options array
+     * Where options array may contain
+     *
+     * - `debugOnly` - boolean - (default: false) Whether or not you want to load the plugin when in 'debug' mode only
+     * - `bootstrap` - boolean - (default: false) Whether or not you want the $plugin/config/bootstrap.php file loaded.
+     * - `routes` - boolean - (default: false) Whether or not you want to load the $plugin/config/routes.php file.
+     * - `ignoreMissing` - boolean - (default: false) Set to true to ignore missing bootstrap/routes files.
+     * - `autoload` - boolean - (default: false) Whether or not you want an autoloader registered
+     */
+    'Plugins' => [
+        'Dogs' => ['autoload' => true, 'bootstrap' => true, 'routes' => true], // a simple Dogs plugin
```

Comment:
Probably a commented `MyPlugin` is more clear than an uncommented `Dogs` :smile: